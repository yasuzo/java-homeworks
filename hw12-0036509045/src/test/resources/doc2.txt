This is sample \\ text. \{$= $}
{$ FOR i 1 10 "DF" $}
 This is {$= i $}-th time this message is generated.
{$EnD  $}
{$For i 0 10 2 $}
 sin({$   =   i$}^2) = {$= i i * @sin "0.000\"" @decfmt $}
{$eND$}

{$ FOR ewe_32 1 10 $}
 This is {$= 32 ew3_32 * @sing_22 $}-th time this message is generated.
{$EnD  $}