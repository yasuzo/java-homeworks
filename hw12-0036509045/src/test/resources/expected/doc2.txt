This is sample \\ text. \{$= $}
{$ FOR i 1 10 "DF" $}
 This is {$= i $}-th time this message is generated.
{$ END $}
{$ FOR i 0 10 2 $}
 sin({$= i $}^2) = {$= i i * @sin "0.000\"" @decfmt $}
{$ END $}

{$ FOR ewe_32 1 10 $}
 This is {$= 32 ew3_32 * @sing_22 $}-th time this message is generated.
{$ END $}